NOTEA/Utilities/ListManipulation/ListManipulator.cs

```diff
@@ -35,7 +35,7 @@ public class ListManipulator
             return temp == null ? null : list => temp(list).ToList();
         }
 
-        private Func<IEnumerable<ConspectModel>, IEnumerable<ConspectModel>> GenerateFilter(string searchBy, string searchValue)
+        public Func<IEnumerable<ConspectModel>, IEnumerable<ConspectModel>> GenerateFilter(string searchBy, string searchValue)
```

Comment:
Is this normal for testing?

Reply:
pamirsau, kad sita keiciau, bet man nerado vis